### ISSUE TITLE
Salmonella infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6194]
**Preferred term label:**
Salmonella infection

**Synonyms**
Salmonellosis

**Definition (free text, please give PubMed ID)**
An infection caused by the bacteria of the salmonella type.
PMID: 28720334 

**Parent term (use hpo.jax.org/app)**
Bacterial infection (Issue #6193)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 615978

**Comment by pnrobinson** [comment]
We would characterize this as a disease and use this Mondo term: https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0000827
